addons/mail/models/mail_mail.py

```diff
@@ -346,6 +348,92 @@ def _filter_mail_mail_servers(self, mail_servers):
             return mail_servers.filtered(lambda server: not server.owner_user_id)
         return mail_servers.filtered(lambda server: server.owner_user_id in [self.env['res.users'], self.mail_message_id.create_uid])
 
+    def _delay_personal_mail_servers(self, mail_server):
```

Comment:
Trying to match split_by_mail_configuration naming, so that we have a coherent naming for methods that globally divide a batch of emails into chunks to handle (and probably to define after the other one to ease reading the code)
```suggestion
    def _split_by_delayed_batch(self, mail_server):
```